services/src/main/java/org/apache/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -219,6 +219,7 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
       String connectionId = getAvaticaProtobufConnectionId(protobufRequest);
       targetServer = hostFinder.findServerAvatica(connectionId);
       request.setAttribute(AVATICA_QUERY_ATTRIBUTE, requestBytes);
+      LOG.debug("Forwarding protobuf JDBC connection [%s] to broker [%s]", connectionId, targetServer);
```

Comment:
Should there be an option to make these newly added debug logs be info level? I've always found the router logs to be basically worthless unless stuff is exploding, and letting it print stuff about what it is forwarding to where without the overhead of the massive amount of debug level logs might be useful.

Reply:
Thinking further about this I think would be better handled if the routing logic was just moved out of `AsyncQueryForwardingServlet` into some dedicated query router class that does the routing logic, then these new logs would live in that class and debug logging could more selectively just enable those logs (without picking up all the underlying jetty servlet debug logs).